assets/training/distillation/src/validate_pipeline.py

```diff
@@ -235,6 +241,9 @@ def _validate_record_for_type_NLU_QA(self, record: list) -> str:
         if self._get_cod_status():
             return f"Chain of density is not supported for task type {DataGenerationTaskType.NLU_QUESTION_ANSWERING}"
 
+        if self._get_max_len_summary():
+            return f"Max length summary is not supported for task type {DataGenerationTaskType.NLU_QUESTION_ANSWERING}"
```

Comment:
maybe we should not fail run even when this parameter is updated buy the end users. We will simply not use it and if required lets throw a warning